src/Microsoft.ML.Core/Data/RoleMappedSchema.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
>using System.Linq [](start = 0, length = 17)

My VS highlight gray using System and using System.Linq for me and suggest to remove them as unused.